retrieval_service/datastore/providers/neo4j_graph_test.py

```diff
@@ -171,48 +171,58 @@ async def test_total_similar_to_relationships_count(ds: neo4j_graph.Client):
         print(count)
 
     # Bi-directional pairs are counted only once
-    expected_count = 217
+    expected_count = 172
```

Comment:
I polished relationships.csv file, causing the deletion of some irrelevant relationships